### PR TITLE
Updated "Our OSS Projects" page

### PR DESCRIPTION
**Pull request by jobyko**
This pull request contains:

1. New page for BeSman with link from "Our OSS Projects" page.
2. Provided Equal space for the projects in "Our OSS Projects" page, #32 
3. Fix for #31 [broken images].
4. Updated 5 more projects under "Tracked OSS projects" page.

### CODE REVIEW

**Comment by jobyko** [comment]
@asa1997 Please share your feedback.

File: docs/our_oss_project.md [hunk 1]
```diff
@@ -9,96 +9,11 @@ hide:
...
-
-<div align="justify">
-BeSman environments are pre-bundled with tools and dependencies for a specific open project and it can be provisioned using simple bash shell commands from BeSman utility.
-
- 
-BeSman utility will also provide configurable capabilities that would permit the user to configure the tools and dependencies to be pre-bundled in the environment.
-
- 
-Individuals will have complete control over what goes into building and provisioning the environment.
+BeSman is a command-line utility to provision customized environments for each open source tech stack.
```
Comment (asa1997) [inline docs/our_oss_project.md:12]: I believe this line is getting repeated in the BeSman's file.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comments

File: docs/our_oss_project.md (hunk 1 above)
Comment (jobyko) [reply docs/our_oss_project.md]: @asa1997  
Updated The project description. Please check and share your comments. 

 @panickervinod  @sumodgeorge

**Review by asa1997 — APPROVED** [review]
@panickervinod @jobyko LGTM

**Pull request merged**